src/main/java/com/minecolonies/coremod/colony/managers/RaidManager.java

```diff
@@ -284,15 +297,15 @@ else if (ShipBasedRaiderUtils.canSpawnShipAt(colony, targetSpawnPoint, amount, s
             {
                 final String biomePath = colony.getWorld().getBiome(targetSpawnPoint).getRegistryName().getPath();
                 final HordeRaidEvent event;
-                if (biomePath.contains(DESERT_BIOME_ID))
+                if (biomePath.contains(DESERT_BIOME_ID) || rand.nextInt(100) < IGNORE_BIOME_CHANCE)
```

Comment:
but this way the egyptian raid is slighly more likely than the other ones. Can't we do a rand.nextInt(100) and then if numb == 1, egyptian, if numb == 2 then amazon, (or even a small range)

Reply:
hm in any case the egyptian would be rolled first

Reply:
but ye we could make it one roll

Reply:
yeah but if egyptian is 1 or 2, amazon is 3 or 4 next is 5 or 6 then the chance is equal

Reply:
check now^^